<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лаборант</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/if_user_white.png</normaloff>../res/if_user_white.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTableView{
	background-color:#ffffff;
	alternate-background-color:#e2e2e2;
	color:#3a3b3a;
	border-style:solid;
	border-color:#ffffff;
	border-width:0px;
	selection-color:#ffffff;
	selection-background-color:#54b0cb;
}
QHeaderView::section{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:40px;
	border: #ffffff;
	border-style:solid;
	border-width:0px;
}
QComboBox{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:50px;
	height:30px;
	font:bold;
	padding-left:5px
}
QComboBox QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;
    border: 2px solid #ffffff;
	padding-left:2px;
	padding-right:2px;
	padding-top:2px;
	padding-bottom:2px
}
QComboBox::drop-down {
   width: 24px;
   border: 2px;
}
QComboBox::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:150px;
	height:30px;
	font:bold;
	padding-left:5px
}
QDateEdit QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;;
}
QDateEdit::drop-down {
   width: 24px;
   border: 2px;
}
QDateEdit::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QMenuBar{
	background-color:#3a3b3a;
	color:#ffffff;
	selection-color:#54b0cb;
	font:bold
}
QMenuBar::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	padding-left:2px
}
QToolBar{
	background-color:#3a3b3a;
	color:#ffffff;
}
QMenu {
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	padding-left:2px;
}
QMenu::hover {
    background-color: #448fa6;
    color:#3a3b3a;
	font:bold
}
QMenu::item {
    background-color:#3a3b3a;
	color:#ffffff;
}
QMenu::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	padding-left:10px;
}
QTabWidget{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTabBar::tab:right:selected{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QTabBar::tab:right:!selected{
    background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QGroupBox{
	background-color:#448fa6;
	color:#ffffff;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QCheckBox{
	background-color:#3a3b3a;
	color:#ffffff;
}
QCheckBox::indicator {
    width: 36px;
    height: 36px;
}
QCheckBox::indicator:unchecked {
    image: url(&quot;./res/if_toggle-left_white.png&quot;);
}
QCheckBox::indicator:checked {
    image: url(&quot;./res/if_toggle-right_white.png&quot;);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tables">
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="4">
         <spacer name="verticalSpacer_15">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="4">
         <spacer name="verticalSpacer_13">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="4">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2017</year>
            <month>10</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="11" column="4">
         <spacer name="verticalSpacer_18">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="4">
         <spacer name="verticalSpacer_19">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="4">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="4">
         <spacer name="verticalSpacer_16">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="4">
         <spacer name="verticalSpacer_20">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="14" column="4">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="3">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="13" column="4">
         <spacer name="verticalSpacer_17">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="4">
         <spacer name="verticalSpacer_14">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4">
         <spacer name="verticalSpacer_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="4">
         <spacer name="verticalSpacer_29">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="editSwitch">
          <property name="text">
           <string>Разрешить редактирование</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="15" colspan="3">
         <widget class="QTableView" name="mainTable">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="complete">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../../Pictures/if — 737373/if_calendar_2561349.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tripEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addTrip"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_log-out_white.png</normaloff>../res/if_log-out_white.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="toolTip">
    <string>Выйти из приложения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="addTrip">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_plus-circle_white.png</normaloff>../res/if_plus-circle_white.png</iconset>
   </property>
   <property name="text">
    <string>Новый рейс</string>
   </property>
   <property name="toolTip">
    <string>Начать новый рейс</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionPrintDoc">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_printer_white.png</normaloff>../res/if_printer_white.png</iconset>
   </property>
   <property name="text">
    <string>printDoc</string>
   </property>
   <property name="toolTip">
    <string>Распечатать ТТН</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
